packages/odyssey/src/scss/components/_tab.scss

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
General Sass notes:

- Can we remove the nesting to keep the resultant CSS as unspecific as possible?
- For names, can we stick with `.ods-tab--*`? It definitely reads more awkwardly here, but something like `.ods-tab--container` and `.ods-tab--list` will ensure we know exactly which component it belongs to.
- Can we include the `tab` component name in the variables?

Reply:
These are all great thoughts. 

- Nesting removed
- Tab class names changed. The class names are now `ods-tabs`, `ods-tabs--tablist`, `ods-tabs--tab` and `ods-tabs--tabpanel`. I know our rule is to refer to components in the singular form, however, it felt bad to me to have a class name called `ods-tab--tab`. I'm happy to defer to you on this.
- Variable names are now `--ods-tabs-indicator-width` and `--ods-tabs-indicator-pos-x`. Should they use the double dash convention? Looks a little weird (e.g `--ods-tabs--indicator-pos-x`/`--ods-tabs--indicator-width`)

Reply:
Agreed on component class naming, and I think your instinct is right about the weirdness of double-dashes in the CSS vars. We can let them have their syntax since it's, in part, determined by the spec.